locales/en/email-strings.ftl

```diff
@@ -90,3 +90,31 @@ email-breach-alert-all-data-points-title = Your exposed data:
 email-breach-alert-all-next-steps-lead = We’ll guide you step-by-step on how to resolve this data breach.
 email-breach-alert-all-next-steps-cta-label = Let’s get started
 email-breach-alert-all-next-steps-button-dashboard = Go to Dashboard
+
+# Monthly Report Free User
+
+# Variables:
+#   $monthOfReport is the month for the report, e.g. July, August.
+email-monthly-report-hero-free-heading = Your { $monthOfReport } monthly { -brand-mozilla-monitor } report
+email-monthly-report-hero-free-body = A summary of what needs your attention, and what’s been resolved over the past month.
+# Variables:
+#   $data_point_count (number) - The number of data breaches that were exposed
+email-monthly-report-free-scan-results-data-points-label = <stat>{ $data_point_count }</stat> Data breaches
```

Comment:
```suggestion
email-monthly-report-free-scan-results-data-points-label =
    {$data_point_count ->
        [one] <stat>{ $data_point_count }</stat> Data breach
       *[other] <stat>{ $data_point_count }</stat> Data breaches
    }
```

This applies to all string with a number of breaches.

Why is the text capitalized in these strings after the number? That feels strange to me.

Reply:
OK, the screenshots were not there when I commented. Boy this is going to break badly, because locales will move the number in the middle of their translations.

The only way to make this work is to remove the number from the string, and add a clear explaining that the text is displayed under a large number. [We have already examples in code.](https://github.com/mozilla/blurts-server/blob/main/locales/en/app.ftl#L117-L133)